client/src/components/fortune.vue

```diff
@@ -113,25 +162,49 @@ export default {
       evt.preventDefault()
       this.$refs.addFortuneModal.hide()
       this.initForm()
+    },
+    removeFortune (fortuneID) {
+      this.$log.debug('Adding new fortune')
+      const path = `http://localhost:8081/fortune/${fortuneID}`
+      this.$log.debug('Calling url: ', path)
+      this.$log.debug('Calling DELETE /fortune/{id}')
+      axios.post(path)
+        .then(() => {
+          this.$log.debug('Delete Fortune!')
+          this.message = 'Fortune Removed!'
+          this.showMessage = true
+          this.$log.debug('Updating fortunes')
+          this.getFortunes()
+        })
+        .catch((error) => {
+          this.$log.error(error)
+          this.getFortunes()
+        })
+    },
+    onDeleteFortune (fortune) {
+      this.$log.debug('Calling Delete Fortune')
+      this.removeFortune(fortune.id)
     }
   },
   created () {
+    this.$log.debug('Fortune Component Created')
     this.getFortunes()
   }
 }
 </script>
 
 <style scoped>
 h1 img {
-  display:inline;
+  display: inline;
 }
-th:nth-child(1){
-   width: 50%;
+th:nth-child(1) {
+  width: 50%;
 }
-th:nth-child(2), th:nth-child(4){
-   width: 20%;
+th:nth-child(2),
+th:nth-child(4) {
+  width: 20%;
 }
-th:nth-child(3){
-   width: 10%;
+th:nth-child(3) {
+  width: 10%;
```

Comment:
Since you're using bootstrap everywhere else you might want to consider using bootstrap here. Also not something to tackle for this PR but have you tested this in mobile?

Reply:
Sent a PR to bootstrap to add support for more class options